internal/command/files.go

```diff
@@ -0,0 +1,43 @@
+package command
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getTestFiles(fileList string, testRunner TestRunner) ([]string, error) {
+	if fileList != "" {
+		return getTestFilesFromFile(fileList)
+	} else {
+		return testRunner.GetFiles()
+	}
+}
+
+func getTestFilesFromFile(path string) ([]string, error) {
```

Comment:
This method is moved unchanged from `command.Run` as it's now used in both `command.Run` and `command.Plan`.